### ISSUE TITLE
Many sites broken

### ISSUE THREAD

**Issue by TexasDex** [uBlockOrigin/uAssets#8031]
### URL(s) where the issue occurs

https://app.slack.com et al

### Describe the issue

After a ubo update and browser restart this morning, all the third party JS is blocked on many sites, apparently by this rule:

*$script,3p,denyallow=9xbud.com|disqus.com|disquscdn.com|jwpsrv.com|plyr.io|jsdelivr.net|twimg.com|twitter.com|fastly.net|fastlylb.net|jquery.com|hwcdn.net|hcaptcha.com|recaptcha.net|cloudflare.com|cloudflare.net|google.com|googleapis.com|g

### Screenshot(s)

NA

### Versions

- Browser/version: FF 81.0.2 (64-bit)
- uBlock Origin version: 1.30.2

### Settings

-No major changes in ubo config that I can think of

### Notes

Looks like this rule was added on commit 4a035b8

**Comment by mapx-** [comment]
it was fixed 9 minutes after that commit (4 days ago), just update your lists

**Comment by TexasDex** [comment]
Ah, gotcha, thanks.  I guess I managed to update at just the wrong time?

**Comment by mapx-** [comment]
the commit fixing the typo => https://github.com/uBlockOrigin/uAssets/commit/82226c595a11b6562488cddfb7beb51e3151f4e0

**Comment by TexasDex** [comment]
Thanks!

**Comment by krystian3w** [comment]
Duplicate of https://github.com/uBlockOrigin/uAssets/issues/8016
